sql/src/main/java/io/crate/planner/projection/TopNProjection.java

```diff
@@ -119,10 +121,13 @@ public boolean isOrdered() {
 
     @Override
     public void prependOutput(Symbol symbol) {
-        InputColumn.shiftRight(outputs);
-        outputs.add(0, symbol);
+        Set<Symbol> symbolsToShift = Sets.newHashSet(outputs);
```

Comment:
pls move down to where it is used. btw. afaik I assigned a local var here by intent for clarification